docs/kqueen.rst

```diff
@@ -52,9 +52,12 @@ Serializers
 Configuration
 --------------
 
-Sample configuration files are located in the `config/` directory. The default configuration file is `config/dev.py`.
-To define a different configuration file, set the `KQUEEN_CONFIG_FILE` environment variable.  To override the values defined
-in the configuration file, set the environment variable matching the KQUEEN_<config_parameter_name> pattern.
+Sample configuration files are located in the ``config/`` directory. The default
+configuration file is ``config/dev.py``.
+To define a different configuration file, set the ``KQUEEN_CONFIG_FILE``
+environment variable. To override the values defined in the configuration file,
+set the environment variable matching the ``KQUEEN_<config_parameter_name>``
+pattern.
```

Comment:
I suggest to add information, that mentioned environment variables are set in the docker-compose file and if you are running kqueen in the docker, all config settings (env variables that are responsible for that) are located in the docker-compose.demo (or production) yaml files in the repository root.